spec/system/order_filter_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Order filtering', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:order1) { create(:order, employee: employee, status: 'delivered') }
```

Comment:
Consider naming those orders: :order_delivered, :order_not_delivered - it may make specs easier to read

Reply:
True, fixed.